planet/scripts/cli.py

```diff
@@ -324,6 +324,11 @@ async def create(ctx, name, ids, bundle, item_type, email, cloudconfig, clip,
         tools=tools)
 
     async with orders_client(ctx) as cl:
-        order = await cl.create_order(request)
+        try:
+            order = await cl.create_order(request)
+        except OrderError as error:
+            service_resp = json.loads(err.__context__.message)
+            LOGGER.info("Order failed: service_resp=%r", service_resp)
+            raise click.ClickException(error.message)
```

Comment:
It seems beneficial to give the users some insight into the cause of the failure without them having to access and sift through logs. It would be helpful to tell them, say, whether they entered bad data for the order creation or they are up against quota limits. I would lean toward giving them the raw service response over making them work for the information. Yes, changing the logging level is an option. It just seems like it should be used for debugging issues that are more infrequent than order creation failing.

Reply:
I think I see what you mean: that the service response details could get lost in info level messages from our dependencies? It's true that urllib3 is super chatty and httpx or aiohttp might also be. In that case, `echo(service_resp, err=True)` could be the way to go. How about I do that for now if `-v` is on and we iterate on the best approach as we go?